selenium_tests/basic_test.py

```diff
@@ -173,3 +180,23 @@ def test_switch_program(self):
         self.wait().until(
             lambda driver: driver.find_element_by_css_selector('.result-info span').text == '{} Results'.format(count)
         )
+
+
+class ProgramPageTests(SeleniumTestsBase):
+    """Look at the program page"""
+
+    def test_program_page(self):
+        """Test viewing the program page"""
+        self.login_via_admin(self.user)
+
+        page = ProgramPageFactory.create(program=self.program, title="A Program Title")
+        for _ in range(3):
+            FacultyFactory.create(program_page=page)
+        for _ in range(3):
+            InfoLinksFactory.create(program_page=page)
+        for _ in range(3):
+            SemesterDateFactory.create(program_page=page)
+        for course in self.program.course_set.all():
+            ProgramCourseFactory.create(program_page=page, course=course)
+
+        self.get("/a-program-title/")
```

Comment:
if we're creating that test data above but not testing for any artifacts from that test data here, why are we creating the test data?